pyvo/registry/rtcons.py

```diff
@@ -22,9 +22,8 @@
 from .import regtap
 
 
-__all__ = ["Freetext", "Author", "Servicetype", "Waveband",
-           "Datamodel", "Ivoid", "UCD", "Spatial", "Spectral", "Temporal",
-           "Constraint", "build_regtap_query", "RegTAPFeatureMissing"]
+# Classes from this module are exposed at the higher level namespace, not listing them here
+__all__ = ["build_regtap_query"]
```

Comment:
I'm slightly concerned about client code whose imports may rely on the old access, but I'm guessing these constraint classes are unlikely to be used directly in much client code, so my concern is pretty small.

Reply:
I think this will only affect those that import `*` from `rtcons` which is an anti-pattern. The module's `__init__` specifies individual classes but it doesn't include `build_regtap_query` whether on purpose or not.

Reply:
Yes exactly. Direct imports will keep working, this is just to avoid having a class exposed at two API locations, and then having multiple API reference entry points.

Reply:
IMO, there are a lot of places where the API could be improved or fixed, or at least consolidated where to expose a class, but this PR only goas as far to fix the inconsistencies that came up through the documentation.